src/core/qgslogger.h

```diff
@@ -101,6 +103,12 @@ class CORE_EXPORT QgsLogger
     //! Goes to qFatal.
     static void fatal( const QString &msg );
 
+    //! Returns true if log level is => than trace level
+    static bool isDebugEnabled() { return 1 <= QgsLogger::debugLevel(); }
+
+    //! Returns true if log level is => than debug level
+    static bool isTraceEnabled() { return 2 <= QgsLogger::debugLevel(); }
```

Comment:
I would also refrain from introducing `isTraceEnabled()` - it gives the impression it is some kind of new logging mode. If would stick to using `if ( QgsLogger::debugLevel() >= 2 )` in the code instead which is more explicit, or introduce something like `static bool isDebugLevelEnabled( int level )`